hs_core/views/__init__.py

```diff
@@ -1854,3 +1834,20 @@ def get_context_data(self, **kwargs):
             'profile_user': u,
             'groups': groups,
         }
+
```

Comment:
Having nearly 2,000 lines of code in this __init__.py is bad and a future refactor will pull these classes out into logical groups. For now, adding 'the ugly way' to have things all in the same place, until cleanup/refactoring.